src/insert/spacefinder/article.ts

```diff
@@ -209,12 +225,7 @@ const addMobileAndTabletInlineAds = (
 			return fillSlot(
 				name,
 				slot,
-				// Add the mobile portrait interstitial size to inline1 and inline2
-				i == 1 || i == 2
-					? {
-							mobile: [adSizes.portraitInterstitial],
-						}
-					: undefined,
+				additionalMobileAndTabletInlineSizes(i),
```

Comment:
Previously we passed undefined to `fillSlot` if there wasn't a match. I assume `{}` will not change the behaviour but it would be good to narrow the expected types if we can.

Reply:
I would argue it's fine accepting both? `undefined` works because it's an optional parameter ([which is defaulted to `{}` eventually](https://github.com/guardian/commercial/blob/jlk/prebid-pubmatic-interscroller/src/define/Advert.ts#L137)) and `{}` satisfies the `SizeMapping` type.

We could require a `SizeMapping` to have at least 1 property which makes sense, but setting that up is a bit of a pain AFAIK? I think we may rely on empty size mappings elsewhere though.

Reply:
It seems like we're using `{}` as a proxy for no sizes in which case 'undefined' is more applicable? Very minor just a suggestion!